client/src/components/GameArea.js

```diff
@@ -1,108 +1,122 @@
-import React, { Component } from "react";
-import PropTypes from "prop-types";
-
-import * as sdk from '../js-sdk/sdk';
-
-import "../styles/GameArea.css";
-import Card from "./Card.js";
-import "../styles/Card.css";
-
-import NavigationBar from './NavigationBar';
-import Sidebar from './Sidebar';
-
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
 import {
   Jumbotron,
   Container,
   Row,
   Col,
   Button,
-  ButtonGroup
-} from "reactstrap";
+  ButtonGroup,
+} from 'reactstrap';
 
-class GameArea extends Component {
-  lordyUrl =
-    "https://api.scryfall.com/cards/1d9d8732-9ff2-42e4-bdfc-723cb6a76969?format=json";
+import Card from './Card.js';
+
+import NavigationBar from './NavigationBar';
+import Sidebar from './Sidebar';
 
+import '../styles/GameArea.css';
+import '../styles/Card.css';
+
+class GameArea extends Component {
   constructor(props) {
     super(props);
+
+    const { cards } = this.props;
     this.state = {
-      player: "Anthony",
       life: 0,
-      top_row: [],
-      bottom_row: [],
+      cards,
     };
 
     this.increment = this.increment.bind(this);
     this.decrement = this.decrement.bind(this);
-    this.toggleCard = this.toggleCard.bind(this);
-  }
-
-  player1Callback(docData) {
-    const { gameActions } = this.props;
-    gameActions.updatePlayer("player1", docData)
   }
 
-  toggleCard(card) {
-    if (card.state.tapped) {
-      sdk.untap(this.props.gameState.gameId, card.id)
-    }
-    else {
-      sdk.tap(this.props.gameState.gameId, card.id)
-    }
+  componentWillReceiveProps(nextProps) {
+    const { cards } = nextProps;
+    this.setState({
+      cards,
+    });
   }
 
   increment(x) {
     this.setState({
-      life: Number(this.state.life) + Number(x)
+      life: Number(this.state.life) + Number(x),
```

Comment:
Updating this to use the real firestore life total probably wouldn't be super hard (as far as I know). But I'm also fine with doing this later.

Reply:
I was thinking about doing that but there needs to be some logic tying the current user to a player record in Firestore, which I think will be easier after we tackle the Firebase Auth stuff